tests/__mocks__/constants.ts

```diff
@@ -0,0 +1,24 @@
+import { STRINGS } from "./strings";
+
+export const AUTHED_USER = {
+  email: STRINGS.UBQ_EMAIL,
```

Comment:
```suggestion
  email: STRINGS.EMAIL,
```

"UBQ" should only be used in reference to the governance token. It is not intended to be used to refer to the organization.